server/container_update_resources.go

```diff
@@ -51,3 +61,52 @@ func toOCIResources(r *pb.LinuxContainerResources) *rspec.LinuxResources {
 		// TODO(runcom): OOMScoreAdj is missing
 	}
 }
+
+func (s *Server) updateContainerResources(c *oci.Container, resources *rspec.LinuxResources) error {
+	// generate updated spec under the <bundlePath>/config.updated.json
+	if err := updateContainerSpec(c, resources); err != nil {
+		return err
+	}
+
+	if err := s.Runtime().UpdateContainer(c, resources); err != nil {
```

Comment:
hm my first intention was to put the `s.Runtime().UpdateContainer(c, resources)` to be the last call under the method, to reduce number of calls to it in case if some previous step failed, but if we ok to change the order of calls like:
```
if err := s.Runtime().UpdateContainer(c, resources); err != nil {}
if err := updateContainerSpec(c, resources); err != nil {
		return err
	}
```
I do not really think we need atomic call or clean up in this case

@haircommander WDYT?